### DISCUSSION TITLE
[Kontoverwaltung] Übersicht Schülerdaten Admin

### DISCUSSION THREAD

**Discussion by Schmarvinius** [SE-TINF22B2/G2-Schoolify#129] [User Stories]
## User Story:
Als Admin möchte ich eine Übersicht der Daten eines Schülers haben.

## Beschreibung
Eine Ansicht in der persönliche Daten stehen wie auch Noten in teilnehmenden Fächern (aller Jahre)

## Akzeptanz-Kriterien
- [ ] Liste persönlicher Daten
- [ ] Liste aller teilgenommenen Jahre/Klassen
- [ ] In der o.g. Liste sollen alle teilgenommenen Fächer mit zugehörigen Noten stehen (veränderbar)

### Issues
- #143

**Comment by laralngn** [comment]
Hier fehlt auch der Nutzen.
